pipelines/config/dependency-builds.yml

```diff
@@ -85,6 +91,7 @@ dependencies:
           - line: 2.2.4X
           - line: 3.0.1X
         removal_strategy: keep_all
+    versions_to_keep: 99
```

Comment:
This is basically `keep_all`. It'll be nice to have a yml templating setup where was can use a variable here in place of this magic number.

Reply:
👍